cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js

```diff
@@ -88,10 +60,25 @@ function processEvent(event, callback) {
     const message = JSON.parse(event.Records[0].Sns.Message);
 
     let messageText;
-    if (message.source == 'aws.config') {
+    var attachments = null;
+
+    if (message.hasOwnProperty('mail')) {
+        const source = message.mail.commonHeaders.from.join();
+        const destination = message.mail.destination.join();
+        const subject = message.mail.commonHeaders.subject;
+        console.log(message.content);
+        messageText = `email from ${source} to ${destination} with subject of ${subject}`;
+        attachments = [
+            {
+             fallback: 'foo',
+             title: 'email content',
+             text: message.content
+            }
+        ];
+    } else if (message.source == 'aws.config') {
         const alarmName = message.detail.configRuleName;
         const newState = message.detail.newEvaluationResult.complianceType;
-        messageText = `${alarmName} state is now ${newState}`
+        messageText = `${alarmName} state is now ${newState}`;
     } else if (message.source == 'aws.ssm') {
```

Comment:
This line confuses me... The code to check for SSM events was already committed? And that is what was deployed to dev? But then you had to make additional changes to deploy it to staging?

Reply:
The code editor wanted a semicolon at the end of the line; I guess I missed that before. The missing semicolon doesn't cause a failure.